app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,22 @@
+class PostsController < ApplicationController
+  def show
+    @post = Post.find(params[:id])
+  end
+
+  def edit
+    @post = Post.find(params[:id])
+  end
+
+  def update
+    @post = Post.find(params[:id])
```

Comment:
I'm not totally sure on this (maybe @jmburges can weigh in), but I wonder if we should actually just have the `update` action be totally empty. force them to build it from the ground up?